src/common/iop_order.c

```diff
@@ -60,7 +60,8 @@ static int _ioppr_legacy_iop_order_step(GList **_iop_order_list, GList *history_
     _ioppr_move_iop_before(_iop_order_list, "colorout", "clahe", dont_move);
     _ioppr_insert_iop_after(_iop_order_list, history_list, "basicadj", "colorin", dont_move);
     _ioppr_insert_iop_after(_iop_order_list, history_list, "rgbcurve", "levels", dont_move);
-
+    _ioppr_insert_iop_after(_iop_order_list, history_list, "lut3d", "velvia", dont_move);
```

Comment:
This may not work as expected and return a NULL work profile. Until is fixed you can change velvia with splittoning.